worker/worker/loader/base.py

```diff
@@ -16,3 +17,5 @@ def __init__(self, file: File, file_bytes: IO[bytes], env: Settings) -> None:
         self.file = file
         self.file_bytes = file_bytes
         self.env = env
+
+        self.host = "localhost" if "pytest" in sys.modules else "unstructured"
```

Comment:
inspired by volkswagen

Reply:
im afraid you are right Will

Reply:
[8e97bbb](https://github.com/i-dot-ai/redbox/pull/917/commits/8e97bbbd5161ede3638867bd1b28f08ba2928b82)